openassessment/workflow/models.py

```diff
@@ -316,8 +318,8 @@ def update_from_assessments(self, assessment_requirements, override_submitter_re
             old_score = sub_api.get_latest_score_for_submission(self.submission_uuid)
             if (
                     not old_score or # There is no recorded score
-                    not old_score.get('staff_id') or # The recorded score is not a staff score
-                    old_score['points_earned'] != new_staff_score['points_earned'] # Previous staff score doesn't match
+                    not self.STAFF_ANNOTATION_TYPE in old_score.get('annotation_types', []) or  # Not a staff score
```

Comment:
This is actually largely a result of a [previous test run](https://travis-ci.org/edx/edx-ora2/builds/180817774), and allowed me to not error out during "do/don't use the updated edx-submissions code" A/B testing I was running. Should I remove it, since it isn't strictly needed?